js/pages/estimation/cca-manager.less

```diff
@@ -0,0 +1,156 @@
+.panel-default>.panel-heading {
```

Comment:
we added less loader not for just a cool extension.
first of all, less means that you follow it's semantics of folding the rules
then, the Component class provides a field `classes`, which is a BEM class names factory, which, in conjunction with the LESS styles should provide easy and safe way to style a component. It's styles should always be local to the component

Reply:
Without proper guidance on how to apply BEM principles and how the LESS loader implements them, it's extremely difficult to just look at a piece of code and understand what the proper way to apply those principles are.  What we were given was 'here's the BEMHelper...here's a .less loader,...now go', which is grossly insufficient on how to apply those tools properly.  

having just started using BEMHelper (via $component.classes('xxx')), I can tell you that it's very challenging to move from the 'global style' approach of applying classes to the 'define each individual element', so what's presented in this PR is what we could get to work following the patterns that are better understood.

I think it would be useful to dedicate a Atlas/WebAPI meeting to walk through BEM, the BEMHelper and LESS syntax, and this is something that should have been done as part of introducing the less loader and BEM helper. So, please hold your scorn when you don't see something is being applied when there's zero documentation or training on how to apply those methods properly.

Reply:
@chrisknoll , even w/o touching BEM, the presence of `.panel-default>.panel-heading` in LESS file looks like the person, who coded it, totally misses the preprocessor idea